### PR TITLE
[stable10] Update changelog for upcoming 10.0.10

### PR DESCRIPTION
**Pull request by PVince81**
This gives us a little overview how much was done during code freeze and got merged last week :-)

@pmaier1

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,48 @@ All notable changes to this project will be documented in this file.
...
+- Bump symfony 3.4.11 to 3.4.12 - [#31912](https://github.com/owncloud/core/issues/31912)
+- Bump paragonie/random_compat v2.0.15 to v2.0.17 - [#32107](https://github.com/owncloud/core/issues/32107)
+### Removed
+### Fixed
+- Fix calendar or reminder insertion error via CalDAV on MacOS - [#32024](https://github.com/owncloud/core/issues/32024)
+- Properly log failed message when token based authentication is enforced - [#31948](https://github.com/owncloud/core/issues/31948)
+- Prevent share access to birthday calendar - [#31882](https://github.com/owncloud/core/issues/31882)
+- Added space in display names of shared calendar/contact - [#31877](https://github.com/owncloud/core/issues/31877)
+- Deleting a user now also properly deletes their external storages and storage assignations - [#32069](https://github.com/owncloud/core/issues/32069)
+- Improve text about loggin in config.sample.php - [#32049](https://github.com/owncloud/core/issues/32049)
```
Comment (phil-davis) [inline CHANGELOG.md]: s/loggin/logging

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,48 @@ All notable changes to this project will be documented in this file.
...
+- Remove sensitive shared_secret data from occ config:list output - [#31997](https://github.com/owncloud/core/issues/31997)
+- Fix file cache update function to properly handle empty string and nulls with Oracle - [#31996](https://github.com/owncloud/core/issues/31996)
+- Fix bogus etag update when propagating etag for federated shares - [#31992](https://github.com/owncloud/core/issues/31992)
+- Display all failed recipients when sending link share email - [#31935](https://github.com/owncloud/core/issues/31935)
+- Lock public link share dialog while processing - [#31928](https://github.com/owncloud/core/issues/31928)
+- AppManager text typo and PHPdoc return tags - [#31918](https://github.com/owncloud/core/issues/31918)
+- Optimize file uploads with PUT method, with custom mtime, use storage instead of view - [#31891](https://github.com/owncloud/core/issues/31891)
+- Optimize file uploads with PUT, don't fetch and update checksum again, reuse the one from part file - [#31768](https://github.com/owncloud/core/issues/31768)
+- Do not throw an error when the same theme enabled twice - [#31783](https://github.com/owncloud/core/issues/31783)
```
Comment (phil-davis) [inline CHANGELOG.md]: ...same theme is enabled twice

**Comment by PVince81** [comment]
I've fixed the typos and added more items in the last commit.

I think now we've got the meat of all that happened during code freeze :-)

**Review by phil-davis — APPROVED** [review]
LGTM

**Pull request merged**